scheduler/plugin/plugin.go

```diff
@@ -79,22 +76,6 @@ func NewFromConfig(ctx context.Context, config PluginConfig) (framework.Plugin,
 	}
 }
 
-func readFromURI(ctx context.Context, u string) ([]byte, error) {
-	uri, err := url.ParseRequestURI(u)
```

Comment:
the problem was eagerly parsing here (not done in dapr where some logic was copied from). This breaks relative paths like `file://../../foo.wasm`. I noticed this running `make profile`

Reply:
ah right. This was not caught by other tests because they were eagerly converting to an absolute `file:///`